lang/java/tools/src/test/java/org/apache/avro/tool/TestConcatTool.java

```diff
@@ -136,6 +137,38 @@ public void testConcat() throws Exception {
   }
 
   @Test
+  @Ignore
+  // theoretically this test should test the append scenario
+  // the problem is, FileSystem.append does not work for local filesystems
+  // (note: or maybe doing dev on Windows is the problem??)
+  // the best way to test is to write to HDFS and compare append output to the
+  // fresh output file generated by the plain concat operation writing to HDFS
```

Comment:
Sure thing. [AVRO-1876](https://issues.apache.org/jira/browse/AVRO-1876) has an issue. (It should also give you an idea about minimal-useful-info if you're up for filing additional issues in the future.)